orc8r/cloud/deploy/orc8r_deployer/docker/run_deployer

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+Print_Help()
+{
+   echo "./run_deployer runs the orc8r deployer container"
+   echo "orc8r deployer contains scripts which enable user to configure, run prechecks"
+   echo "install, upgrade, verify and cleanup an orc8r deployment"
+   echo
+   echo "Syntax: run_deployer [-d|-r|-b|-h]"
+   echo "options:"
+   echo "-h     Print this Help"
+   echo "-d  deployment dir containing configs and secrets"
+   echo "-r  magma root directory"
+   echo "-b  build the deployer container"
+   echo "note: -deploy_root is mandatory"
+   echo "example: ./run_deployer -deploy_root /tmp/orc8r_14_deployment"
+   echo
+}
+
+if (( $# < 2 )); then
+    # TODO: print usage
+    Print_Help
+    exit 1
+fi
+
+DOCKER_BUILD=false
+while [ -n "$1" ]; do # while loop starts
+	case "$1" in
+	-d)
+        DEPLOY_WORKDIR="$2"
+        shift
+        ;;
+	-r)
+        MAGMA_ROOT="$2"
+        shift
+        ;;
+    -b)
+        DOCKER_BUILD=true
+        shift
+        ;;
+    -h)
+        Print_Help
+        exit;;
+
+	*) echo "Option $1 not recognized" ;;
+	esac
+    shift
+done
+
+if [[ ! -d $DEPLOY_WORKDIR ]]
+then
+    echo "Error ${DEPLOY_WORKDIR} not a directory."
+    exit
+fi
+
+MAGMA_ROOT=${PWD/\/orc8r\/cloud\/deploy\/orc8r_deployer\/docker}
```

Comment:
This assumes a cwd, which causes unexpected errors when users think it's safe to run from e.g. the parent dir. Just use `mroot=${MAGMA_ROOT:-~/magma}`

Reply:
Your comment here https://github.com/magma/magma/pull/5548#discussion_r595779720  suggested otherwise. 
![Screen Shot 2021-04-01 at 5 14 30 AM](https://user-images.githubusercontent.com/8224854/113292283-24d50280-92a9-11eb-84f5-7105c6418cbc.png)
I added a print to dump the magma_root for users to know what the root is getting set as. It can cause unexpected errors when users run it from parent dir, I can add a warning in case it catches users unaware of this behavior. Above form   `mroot=${MAGMA_ROOT:-~/magma}`also might cause errors if users don't typically create magma in their home directory. I will leave this as is. Let me know if this makes sense.